app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>All users</h1>
\ No newline at end of file
+<div>
+<ul class="users-list">
+<% @users.each do |user| %>
+  <%= render partial: "partials/user", locals: {user: user} %>
+<% end %>
+</ul>
+</div>
```

Comment:
### Your Home Page

![image](https://user-images.githubusercontent.com/93039359/230124960-31550b3d-fe8c-410e-8c82-94698e4364a0.png)

### Wireframe Home Page
![image](https://user-images.githubusercontent.com/93039359/230126290-30f391ea-013c-40c5-8cab-845eca262e23.png)

**[OPTIONAL]**

- [ ] Your index page is not similar to the wireframe given to you because if you see then the image is in the left corner and everything else is in the right corner :+1: Kindly follow the wireframe to make your project professional :+1: